static/src/js/operations/queries/getNonNasaDraftUserAcls.js

```diff
@@ -0,0 +1,12 @@
+import { gql } from '@apollo/client'
+
+// Query to get non nasa draft user acls
```

Comment:
This matches matches the ACL, but not really true anymore as they can create and publish concepts now.   Make a comment here.